content/blog/2025/12/2025-12-03-fosdem-presence.md

```diff
@@ -0,0 +1,68 @@
+
+The hackathon is first and foremost about having fun and learning. We want to know what you worked on, what worked, what you struggled with, and what surprised you. We'll conclude the day by asking teams to summarize their day to others. We would like to livestream the sessions on our website, but you can opt-out if you're not comfortable with it.
+
+After the project presentations, we will stay in HSBXL in the evening for a public Matrix Lounge Evening. We're inviting the entire Matrix community to spend a relaxed evening with food, drinks and interesting discussions about Matrix! We're asking all hackathon attendees to [register](https://pretix.eu/matrix/fosdem26-hackathon/) so we can best organize the event.
+
+🕐️ Friday 30, 09:00 - 17:00 CET (local time)  
+📹 Project presentations at 17:00 CET (local time)  
+🛋️ Public Matrix Lounge Evening starting from 18:00 CET (local time)  
+🏢 [HSBXL](https://hsbxl.be/enter/), Rue Osseghem 53, 1080 Molenbeek  
+🍕 Snacks, cold drinks, lunch and dinner included  
+🧑‍💻 For hackers, mentors, artists, project managers & the entire community  
+🎫 Free but [mandatory registration here](https://pretix.eu/matrix/fosdem26-hackathon/)  
+💰 Sponsors [welcome](mailto:events-wg@foundation.matrix.org?subject=Hackathon%20Sponsorship)
+
+## Saturday, 31st January & Sunday, 1st February - Booth
+
+Once more, we are grateful to the FOSDEM organizers for granting us a booth for the whole weekend! If you want to come talk to the team, share your gripes and ideas or come see cool demos, this is your opportunity to do so! We will bring merch and stickers for the community to help us spread the word.
+
+![A group of people are talking to one another on each side of a table. Someone is showing a problem on their phone.](/blog/img/fosdem-25-booth.webp)
+
+We're looking for volunteers to run the booth with us. This includes talking to the community, sharing project news, and distributing merch. Don't worry if it's your first time: We have a booth handbook ready for volunteers and want to limit the time commitment to 2 hours per day.
+
+Reach out to <events-wg@foundation.matrix.org> or in [#events-wg:matrix.org](https://matrix.to/#/#events-wg:matrix.org) if you're interested in staffing the booth with us! We will work out together which slot works best for you.
+
+🕐️ Saturday 1 to Sunday 2, 09:00 - 18:00 CET (local time)  
+🏢 Desk to be announced [on FOSDEM's website](https://fosdem.org/2026/stands/)
+
+## Sunday, 1st February - Devroom
+
+This year FOSDEM organisers tried to accommodate as many projects as possible by asking devroom managers to focus on themes more than on specific projects. They generously granted us a full day devroom on Sunday 1st February to talk about Decentralised Communication, and we're delighted to have received so many proposals!
+
+![A room full of people how are listening to a speaker who is off the frame.](/blog/img/fosdem-25-booth.webp)
```

Comment:
Hm we used this image twice in the same post here. Do we have another option for one of those cases? Its a bit weird to use it twice in the same post.

Reply:
```suggestion
![A room full of people are listening to a speaker who is off the frame.](/blog/img/fosdem-25-booth.webp)
```